packages/core/src/App/Containers/Modals/trading-experience-modal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Button, Icon, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+
+const TradingExperienceModal = ({
+    cfd_score,
+    is_trading_experience_incomplete,
+    setShouldShowTradingAssessmentModal,
+    should_show_trading_assessment_modal,
+    setShouldShowTradeAssessmentForm,
+}) => {
+    React.useEffect(() => {
+        if (cfd_score === 0) {
+            setShouldShowTradingAssessmentModal(true);
```

Comment:
setShouldShowTradingAssessmentModal(cfd_score === 0) instead of this condition

Reply:
Resolved 🫡